src/Agent.Worker/ExecutionContext.cs

```diff
@@ -621,6 +621,8 @@ private void JobServerQueueThrottling_EventReceived(object sender, ThrottlingEve
                     uriBuilder.Path += (Variables.System_TFCollectionUrl.EndsWith("/") ? "" : "/") + "_usersSettings/usage";
                     query["tab"] = "pipelines";
                     query["queryDate"] = queryDate;
+                    query["keywords"] = this.Variables.Build_Number;
```

Comment:
The Azure Pipelines Agent can also run releases, in addition to builds. From the agent side, either only the build variables or the release variables will be defined at a given point in time. I'd consider the release case by doing something like the following snippet:

```cs
if (!string.IsNullOrEmpty(this.Variables.Build_DefintitionName))
{
    query["definition"] = this.Variables.Build_DefinitionName;
    query["keywords"] = this.Variables.Build_Number;
}
else if (!string.IsNullOrEmpty(this.Variables.Release_ReleaseName)
{
    query["definition"] = this.Variables.Release_ReleaseName;
    query["keywords"] = this.Variables.Release_ReleaseId;
}
```